Servers/UI/OJS.Servers.Ui/ClientApp/src/components/guidelines/trees/Tree.module.scss

```diff
@@ -1,8 +1,11 @@
 @use 'src/styles/fonts';
+@use 'src/styles/colors';
 
 .Tree {
+  @extend %font-family-lato;
+  
+  color: colors.$text-dark-gray-color;
   cursor: pointer;
-  font-family: fonts.$font-family-lato-regular, 'Helvetica', 'Arial', sans-serif;
   font-size: 1rem;
   font-weight: 400;
   line-height: 1.5;
```

Comment:
This is not a good practice. `font-family-lato` should be a mixing that takes params for `font-size`, `font-weight` and `line-height`, and generates a shorthand property. Please log it for future fix

Reply:
https://github.com/SoftUni-Internal/exam-systems-issues/issues/517